arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1936 @@
+      "type": "object",
+      "properties": {
+        "sasTtlAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "connectionString": {
+          "description": "Account key credentials for storage account selected by customer for uploading files",
+          "type": "string"
+        },
+        "containerName": {
+          "description": "Root container name where all files will be uploaded",
+          "type": "string"
+        }
+      }
+    },
+    "MessagingEndpointProperties": {
+      "description": "Contains the user facing view of the properties of the Messaging Endpoints used by this IoT Hub",
+      "type": "object",
+      "properties": {
+        "lockDurationAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "ttlAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "maxDeliveryCount": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "CloudToDeviceProperties": {
+      "description": "Contains user facing view of Iot Hub Cloud-To-Device messaging properties",
+      "type": "object",
+      "properties": {
+        "maxDeliveryCount": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "defaultTtlAsIso8601": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "feedback": {
+          "$ref": "#/definitions/FeedbackProperties",
+          "description": "Gets the feedback properties."
+        }
+      }
+    },
+    "OperationsMonitoringProperties": {
+      "description": "Contains the user facing view of the Operation Monitoring properties",
+      "type": "object",
+      "properties": {
+        "events": {
+          "type": "object",
+          "additionalProperties": {
+            "enum": ["None", "Error", "Information"],
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SharedAccessAuthorizationRule": {
+      "type": "object",
+      "properties": {
+        "KeyName": {
+          "type": "string"
+        },
+        "PrimaryKey": {
+          "type": "string"
+        },
+        "IssuerName": {
+          "type": "string"
+        },
+        "SecondaryKey": {
+          "type": "string"
+        },
+        "ClaimType": {
+          "type": "string"
+        },
+        "ClaimValue": {
+          "type": "string"
+        },
+        "Rights": {
+          "type": "array",
+          "items": {
+            "enum": ["Manage", "Send", "Listen", "ManageNotificationHub"],
+            "type": "string"
+          }
+        },
+        "CreatedTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "ModifiedTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "Revision": {
+          "format": "int64",
```

Comment:
Missing description. (Descriptions are important, they are used to generate the help documentation for the APIs)